src/java/org/apache/cassandra/io/sstable/CQLSSTableWriter.java

```diff
@@ -618,6 +654,11 @@ public CQLSSTableWriter build()
                     Types types = createTypes(keyspaceName);
                     Schema.instance.transform(SchemaTransformations.addTypes(types, true));
                     tableMetadata = createTable(types);
+
+                    Indexes indexes = createIndexes(tableMetadata);
```

Comment:
I need non-null `tableMetadata` first to create `Indexes`. So then I have to unbuild that to set them to these metadata afterwards.